src/main/java/com/ericsson/gerrit/plugins/eiffel/events/models/Submitter.java

```diff
@@ -0,0 +1,38 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.gerrit.plugins.eiffel.events.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Submitter {
```

Comment:
This is called "Data.Author" in protocol,, shouldn't it be named in same way as the other protocol handling classes.

Reply:
No, it's different in different events. In EiffelSourceChangeSubmittedEvent its submitter, not author.

Reply:
ok.